docs/monitoring.md

```diff
@@ -267,12 +267,23 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <br>
     <code>?status=[completed|running]</code> list only applications in the chosen state.
     <br>
-    <code>?minDate=[date]</code> earliest date/time to list.
+    <code>?minDate=[date]</code> earliest start date/time to list.
+    <br>
+    <code>?maxDate=[date]</code> latest start date/time to list; uses same format as <code>minDate</code>.
+    <br>
+    <code>?minEndDate=[date]</code> earliest end date/time to list.
+    <br>
+    <code>?maxEndDate=[date]</code> latest end date/time to list; uses same format as <code>minEndDate</code>.
+    <br>
+    <code>?limit=[limit]</code> limits the number of applications listed.
```

Comment:
I think the old version, those "Examples" were meant to specifically refer to `minDate`, which is why `maxDate` said "use the same format as minDate".  Your reorganization is fine (probably better), but in that case, I think you should remove all of the "use the same format as ...", since the referenced param doesn't actually describe anything more.